pkg/minikube/out/out.go

```diff
@@ -359,6 +360,29 @@ func DisplayError(msg string, err error) {
 	ErrT(style.Empty, "")
 	ErrT(style.Sad, "minikube is exiting due to an error. If the above message is not useful, open an issue:")
 	ErrT(style.URL, "https://github.com/kubernetes/minikube/issues/new/choose")
+	logFileName, err := getLatestLogFileName()
+	if err != nil {
+		klog.Warning(err)
+		return
+	}
+	ErrT(style.Tip, "If you are able to drag and drop the following log-file into the issue, we'll be able to make faster progress: {{.logFileName}}", V{"logFileName": logFileName})
+}
+
+func getLatestLogFileName() (string, error) {
+	dir := os.Getenv("TMPDIR")
+	if dir == "" {
+		dir = "/tmp/"
+	}
+	args := fmt.Sprintf("cd %s && echo %s$(ls -t *minikube_*_*_*log | head -1)", dir, dir)
```

Comment:
this wont work on windows.

how about we point them to the log file we store in the minikube home folder ? 

ideally if there is a trick to Open the Issue with that file attached would best

Reply:
We could just always point them to the start file, but what if `minikube addon enable ingress` fails? Should we still point them to the start logs?

Reply:
Also, I don't think there is anyway to create a URL that will attach the file. Sounds like that would be a large security threat if it was a thing.

Reply:
good point, how about we detect what command is failing, if it is not start, we will look for latest temp files.

regardless,  this command wont work windows, can you make it a Go way of finding latest temp files that has word minikube in their file name?

Reply:
> how about we detect what command is failing, if it is not start, we will look for latest temp files
Can do that for sure

> can you make it a Go way of finding latest temp files
Will look into how to do that

Reply:
Done